packages/main/src/themes/TableRow.css

```diff
@@ -90,4 +90,26 @@
 :host([selected][type="Active"]) .ui5-table-row-root:hover:not(:active),
 :host([selected][mode="SingleSelect"]) .ui5-table-row-root:hover:not(:active) {
 	background-color: var(--sapList_Hover_SelectionBackground);
-}
\ No newline at end of file
+}
+
+/* Navigated */
+:host([navigated]) tr {
+	position: relative;
+}
+:host([navigated]) tr:after {
+    display: block;
+    content: '';
+	width: 100%;
+	position: absolute;
+	inset: 0px;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	background: transparent;
+	border-inline-end: 3px solid var(--sapList_SelectionBorderColor);
+	box-sizing: border-box;
+	pointer-events: none;
+	margin-top: -0.5px;
+	margin-bottom: 0.5px;
```

Comment:
This is redundant.

Reply:
Removed it, searching for another solution.